packages/core/botpress/src/util.js

```diff
@@ -123,6 +127,49 @@ const getPackageName = pkg => {
   }
 }
 
+const versionExists = version => {
+  if (version == undefined) {
```

Comment:
Definitely minor, but it's more conventional to use non-strict comparison with `null`